Microsoft.Toolkit.Uwp.UI.Controls/MarkdownTextBlock/Parse/Blocks/TableBlock.cs

```diff
@@ -0,0 +1,334 @@
+
+                    // Ignore any whitespace at the end of the cell.
+                    if (endOfCell > startOfCellContent)
+                    {
+                        while (Common.IsWhiteSpace(markdown[pos - 1]))
+                        {
+                            pos--;
+                        }
+                    }
+
+                    int endOfCellContent = pos;
+
+                    if (endOfLineFound == false || endOfCellContent > startOfCellContent)
+                    {
+                        // Parse the contents of the cell.
+                        contentParser(startOfCellContent, endOfCellContent);
+                    }
+
+                    // End of input?
+                    if (pos == maxEndingPos)
+                    {
+                        break;
+                    }
+
+                    // Move to the next cell, or the next line.
+                    pos = endOfCell + 1;
+
+                    // End of the line?
+                    if (endOfLineFound)
+                    {
+                        break;
+                    }
+                }
+
+                return pos;
+            }
+
+            /// <summary>
+            /// Called when this block type should parse out the goods. Given the markdown, a starting point, and a max ending point
+            /// the block should find the start of the block, find the end and parse out the middle. The end most of the time will not be
+            /// the max ending pos, but it sometimes can be. The function will return where it ended parsing the block in the markdown.
+            /// </summary>
+            /// <returns>the postiion parsed to</returns>
+            internal int Parse(string markdown, int startingPos, int maxEndingPos, int quoteDepth)
+            {
+                Cells = new List<TableCell>();
+                return ParseContents(
+                    markdown,
+                    startingPos,
+                    maxEndingPos,
+                    quoteDepth,
+                    requireVerticalBar: true,
+                    contentParser: (startingPos2, maxEndingPos2) =>
+                    {
+                        var cell = new TableCell();
+                        cell.Inlines = Common.ParseInlineChildren(markdown, startingPos2, maxEndingPos2);
+                        Cells.Add(cell);
+                    });
+            }
+        }
+
+        /// <summary>
+        /// Represents a cell in the table.
+        /// </summary>
+        public class TableCell
+        {
+            /// <summary>
+            /// Gets or sets the cell contents.
+            /// </summary>
+            public IList<MarkdownInline> Inlines { get; set; }
+        }
+    }
+
+    /// <summary>
+    /// The alignment of content in a table column.
+    /// </summary>
+    public enum ColumnAlignment
+    {
+        /// <summary>
+        /// The alignment was not specified.
+        /// </summary>
+        Unspecified,
```

Comment:
Wouldn't 'Unspecified' really mean 'Left' ? And if so do we need this? Couldn't we just reuse the HorizontalAlignment property then?